packages/blockchain-wallet-v4-frontend/src/config/index.js

```diff
@@ -1,13 +1,15 @@
 import Bitcoin from 'bitcoinjs-lib'
 import BitcoinCash from 'bitcoinforksjs-lib'
 
+const isTestnet = process.NODE_ENV === 'testnet'
```

Comment:
Correct is process.env.NODE_ENV, also this doesn't seem to work even if it's correct

Reply:
Apologize for my late night stupidness. This will not work because process.env.NODE_ENV will not be known by the client. I will address it.